server/app.js

```diff
@@ -42,8 +42,6 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
-require("./utils/socket")(server, app);
-
```

Comment:
This is part of the bug fix on the back end. Removing these lines removes the JWT errors and now users can sign in or sign up